Assets/MixedRealityToolkit-Examples/Demos/StandardShader/Scripts/Editor/MaterialMatrixEditor.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.StandardShader
+{
+    /// <summary>
+    /// Editor to build a matrix of spheres demonstrating a spectrum of material properties.
+    /// </summary>
+    [CustomEditor(typeof(MaterialMatrix))]
+    public class MaterialMatrixEditor : Editor
```

Comment:
This may not work as expected because of the custom assembly definitions we're using.

Reply:
Nvm, I see you added one. Great!